### PR TITLE
Add additional integration instructions

### PR DESCRIPTION
**Pull request by pearsonhenri**
Updated to add additional context on how/when to call Inspetor client library. Still some more work to be done, but just wanted to get some initial eyes on this from @phsanzovo, @lbiselli, and @theosato

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -50,17 +43,99 @@ $inspetor = new InspetorClient($inspetor_config);
 ?>
 ```
 
-To setup an Inspetor Library in any language you first need to set some configuration. 
+The first step of integrating Inspetor's antifraud services into your product is instantiating the library.
+
+In order to do so, you will need to pass the `InspetorClient` your product's **App ID** and **Tracker Name**, both of which will be provided to you by the Inspetor Team.
+
+_Et voilà_! You now have an Inspetor client object that is capable of sending all of the information necessary to teach Inspetor's decision layer how to prevent fraud at your company. However, there is obviously a difference between instantiating a client instance and using it properly.
```
Comment (phsanzovo) [inline source/index.html.md]: In 'However, there is obviously a difference between instantiating a client instance and using it properly.' would remove 'obviously'

File: source/index.html.md [hunk 2]
```diff
@@ -50,17 +43,99 @@ $inspetor = new InspetorClient($inspetor_config);
...
+You can think of the relationship between those entities something like this: If a user purchases tickets for a show on your site, Inspetor interprets the action as:
+
+- the creation of a new <a href="#sale">*Sale*</a>
+- the association of that Sale with an existing <a href="#account">*Account*</a>
+- the association of that Sale with an <a href="#item">*Item*</a>
+- the association of that Item with an existing <a href="#item">*Event*</a>
+
+## When to send events to Inspetor
+
+The primary Inspetor entities are **stateful** objects. They have properites that can be updated, and the values of these properties are crucial to our evaluation of transaction validity. However, if we base our evaluation upon outdated or incorrected information, the accuracy of our evaluation is likely to suffer as well. As such, it is critical that you notify Inspetor of any changes to these properties whenever they occur via the Inspetor Client Library.
```
Comment (phsanzovo) [inline source/index.html.md:79]: **critical** (add highlight)

File: source/index.html.md [hunk 3]
```diff
@@ -50,17 +43,99 @@ $inspetor = new InspetorClient($inspetor_config);
...
+
+## Where to insert Inspetor collection functions
+Where in your code base does the Inspetor library belong? Frontend? Backend? Loaded on the site?
+
+Fortunately for you, the answer is quite simple at the moment--we only have one version (PHP) of the Client Library implemented! That means you'll need to integrate our library with your PHP application server.
+
+In the future however, our suggested integration model will be:
+
+- Full event coverage within the server/API level of your application (backend): This allows us to know everything about what is hapening within your product.
+- 1:1 corresponding coverage within your frontend interfaces (Desktop, iOS, and Android): This allows us to enrich the knowledge we're gathering from your server-level transactions with additional information that improves our protection capabilities.
```
Comment (phsanzovo) [inline source/index.html.md]: replace Desktop with WEB

File: source/index.html.md [hunk 4]
```diff
@@ -803,7 +879,7 @@ All the properties can be defined and accessed through **gets** and **sets**
 
 Property | Required | Type | Description
 -------- | -------- | ---- | -----------
-**id** | Yes | String | The id of the payment. This id is the on eyou send to the bank 
+**id** | Yes | String | The id of the payment. This id is the on eyou send to the bank
```
Comment (phsanzovo) [inline source/index.html.md]: on eyou?

**Review by phsanzovo — COMMENTED** [review]
Looks dp cl - like the opening explanations

File: source/index.html.md (hunk 2 above)
Comment (pearsonhenri) [reply source/index.html.md:79]: I have two pretty aggressively highlighted warnings below that do the same

**Review by theosato — APPROVED** [review]

**Pull request merged**